libraries/management/commands/import_library_versions.py

```diff
@@ -0,0 +1,137 @@
+import djclick as click
+
+from libraries.github import GithubAPIClient, GithubDataParser, LibraryUpdater
+from libraries.models import Library, LibraryVersion
+from versions.models import Version
+
+
+@click.command()
+@click.option("--token", is_flag=False, help="Github API token")
+@click.option("--release", is_flag=False, help="Boost version number (example: 1.81.0)")
+def command(release, token):
+    """Cycles through all Versions in the database, and for each version gets the
+    corresponding tag's .gitmodules.
+
+    The command then goes to the same tag of the repo for each library in the
+    .gitmodules file and uses the information to create LibraryVersion instances, and
+    add maintainers to LibraryVersions.
+
+    Args:
+        token (str): Github API token, if you need to use something other than the setting.
+        release (str): Boost version number (example: 1.81.0). If a partial version number is
+        provided, the command process all versions that contain the partial version number
+        (example: "--version="1.7" would process 1.7.0, 1.7.1, 1.7.2, etc.)
+    """
+    client = GithubAPIClient(token=token)
+    parser = GithubDataParser()
+    updater = LibraryUpdater(client=client)
+
+    skipped = []
+
+    if release is not None:
+        versions = Version.objects.filter(name__icontains=release)
+    else:
+        versions = Version.objects.active()
+
+    for version in versions:
+        click.echo(f"Processing version {version.name}...")
+
+        # Get the .gitmodules for the version using the version name, which is also the git tag
+        ref = client.get_ref(ref=version.name)
+        try:
+            raw_gitmodules = client.get_gitmodules(ref=ref)
+        except Exception:
+            # Only happens for one version; uncertain why.
```

Comment:
Is the exception relevant in this case? Would we benefit from printing the exception name or any other related information?

Reply:
@nessita Yeah I was being a little lazy here. The error is a 422, but for the life of me I could not figure out what the actual problem was on GitHub's end and just forgot to import the actual error.

Reply:
@nessita I caught the specific exception and added that reason to the skipped reasons.